driver/ppm_events_public.h

```diff
@@ -23,8 +23,7 @@ or GPL2.txt for full copies of the license.
 /*
  * Macros for packing in different build environments
  */
-
-#if defined(_WIN64) || defined(WIN64) || defined(_WIN32) || defined(WIN32)
+#if !defined(CYGWING_AGENT) && (defined(_WIN64) || defined(WIN64) || defined(_WIN32) || defined(WIN32))
```

Comment:
Can you elaborate more on this exclusion?
Also, I'm still not sure about this `packed` definition, that iirc was introduced for Windows support as well (when compiling with Visual Studio though). Shouldn't the pragma directives be positioned *around* the struct definition, instead of just appended? (e.g. see https://stackoverflow.com/a/3312896)

Reply:
Yes, when building sysdig for CYGWIN environment on Windows _WIN32 env is also true. But we want the normal packed definition in that case, not the one used for Visual Studio.

I'm not sure we even care about the packed definition for Visual Studio builds since we don't have any driver interaction in Windows, but you may be right about the macro not doing anything.